### PR TITLE
Accept applicant page

### PR DESCRIPTION
**Pull request by tanaya09**
## What does this pull request include?
1. UI for accept applicant

## Any outstanding issues in this pull request?
No

## Checklist

- [x] Test locally
- [x] Assign a reviewer
- [ ] ZenHub

### CODE REVIEW

**Review by austinhuang0131 — CHANGES_REQUESTED** [review]
should also reveal their email address for contact (this was mentioned at the end of last demo), same should also be done for the applicant

otherwise ok

**Comment by tanaya09** [comment]
I’ve addressed the requested changes:
Improved the confirmation UI for the applicant page.
Added an email display for the selected applicant. 
Please let me know if further adjustments are needed.

File: Time_Banking/Time_Banking/templates/profile_info.html [hunk 1]
```diff
@@ -20,11 +20,17 @@
                         <img src="{% static 'images/default-avatar-profile-icon-social-600nw-1677509740.jpg.webp' %}" alt="Default Profile Picture" class="profile-picture">
                     {% endif %}
                     <p> {{ user.title }} {{ user.name }}</p>
+                    
                     <p><i class="fa fa-map-marker"></i> {{ user.location }}</p>
                     <p><i class="fa fa-calendar"></i> Joined in {{ user.date_joined|date:"F Y" }}</p>
                     <p>Bio: {{ user.bio }}</p>
+<p><i class="fa fa-envelope"></i> {{ user.email }}</p>
```
Comment (austinhuang0131) [inline Time_Banking/Time_Banking/templates/profile_info.html:27]: not necessary, my point was to only provide email after being accepted, but the person who applied to the service should be able to see the email of the provider on applied_services as well

File: Time_Banking/Time_Banking/templates/view_applicants.html [hunk 2]
```diff
@@ -19,11 +19,14 @@ <h4 class="confirmation-title">Selection confirmed successfully!</h4>
                         Your chosen applicant: 
                         <a href="{% url 'profile_info' user_id=response.user_id %}" class="applicant-link">{{ response.user }}</a>
                     </p>
+                    <p class="confirmation-detail">
+                        <i class="fa fa-envelope"></i> {{ response.user.email }}
+                    </p>
```
Comment (austinhuang0131) [inline Time_Banking/Time_Banking/templates/view_applicants.html:24]: ok

**Review by austinhuang0131 — CHANGES_REQUESTED** [review]

**Pull request merged**